ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -830,7 +834,7 @@ void postCompletionCleanup() {
 
     @Override
     GetOptions getGetOptions() {
```

Comment:
just noting here for reviewers that for subsequent chunks, we already and always pass `Include_All` regardless of what the `GetOptions` are to avoid a situation where some chunks get retrieved and then subsequent chunks end up with Blob_Deleted or Blob_Expired. As long as the metadata chunk is successfully retrieved we ignore the expired/deleted state for subsequent chunks.